test/Player-test.js

```diff
@@ -0,0 +1,34 @@
+const chai = require('chai');
+const expect = chai.expect;
+const Player = require('../js/Player.js');
+const Question = require('../js/Question.js');
+const data = require('../js/data.js');
+
+
+describe('Player', function() {
+  let player; 
+  let question;
+
+  beforeEach(function() {
+    player = new Player('name');
+    question = new Question(data.clues[0]);
```

Comment:
Since there are no promises that `data.clues[0] ` will always have the answer of 'golf' (as used in the first test), maybe we should create a Question explicitly from that first clue in the dataset and use it, so that we know the Question we are using will always be that one.